packages/components/src/templates/next/components/internal/Link/Link.tsx

```diff
@@ -111,46 +15,47 @@ const linkStyles = tv({
 
 const fvHighlightLinkStyles = tv({
   extend: focusVisibleHighlight,
-  base: "text-link outline-none outline-0",
+  base: "outline-none outline-0",
   variants: {
     showExternalIcon: {
       true: `after:content-['_↗']`,
     },
-    isHovered: {
-      true: "text-link-hover underline underline-offset-4",
-    },
   },
 })
 
-export interface LinkProps
-  extends BaseLinkProps,
-    VariantProps<typeof linkStyles> {
-  withFocusVisibleHighlight?: boolean
-}
-
-export function Link({
+export const Link = ({
+  href,
+  current,
+  isDisabled,
+  isExternal,
+  isWithFocusVisibleHighlight,
```

Comment:
should this default to true (for a11y)? only false if overriding with something else perhaps. Then all the links don't need to care

Reply:
Hmm I was thinking for future cases, usages of the Link will be more customised rather than a direct text link? So as long as BaseParagraph is used for text content, it will automatically come with the focus visible stuff.